tests/components/fastdotcom/test_sensor.py

```diff
@@ -0,0 +1,23 @@
+"""Tests for the Fast.com sensor."""
```

Comment:
Adding these tests are not related to this PR and should be added in a preliminary PR before getting to this one.
Also adding the coordinator should be done in it's own PR and not as part of introducing a config flow.

Please split it out. Thanks